web/themes/custom/cloudy/templates/taxonomy/taxonomy-term.html.twig

```diff
@@ -28,6 +28,8 @@
   {% set vocabulary = term.bundle|join %}
   {% if 'community_actions' in vocabulary %}
     {% set vocabulary = 'service type'|t|title %}
+  {% elseif vocabulary == 'neighborhood' %}
+    {% set vocabulary = content.field_neighborhood_type %}
```

Comment:
Hmm. I feel like this is going to include the field template and include some divs we don't want. You should be able to call something like `content.field_neighborhood_type.0.value` to get a string with only the value and nothing else. We can do this in the follow up story where it is easier to review.